src/scalafix/scala/fix/ImplicitToStringCallOnCaseClass.scala

```diff
@@ -0,0 +1,79 @@
+package fix
+
+import scalafix.v1._
+
+import scala.meta._
+
+/**
+ * Lints on string interpolation where its variable part contains `case class` without `toString`.
+ */
+class ImplicitToStringCallOnCaseClass extends SemanticRule("ImplicitToStringCallOnCaseClass") {
```

Comment:
`Implicit` ではなく `Unoverriden` という方が正しいと思います (コメントやファイル名などにも同じことが言えそうです)

```suggestion
class UnoverridenToStringCallOnCaseClass extends SemanticRule("UnoverridenToStringCallOnCaseClass") {
```